domain/users/user_dto.go

```diff
@@ -0,0 +1,67 @@
+package users
+
+import (
+	"moku-moku/utils/errors"
+	"regexp"
+	"strings"
+)
+
+type User struct {
+	Email      string `json:"email"`
+	Username   string `json:"username"`
+	Name       string `json:"name"`
+	Surname    string `json:"surname"`
+	Biography  string `json:"biography"`
+	Birthday   string `json:"birthday"`
+	Password   string `json:"-"`
+	PasswordR  string `json:"-"`
+	ProfilePic string `json:"profile_picture"`
+	Points     int32  `json:"points"`
+}
+
+func (user *User) EmailValidation() *errors.RestErr {
+	// Email sanitation
+	user.Email = strings.TrimSpace(strings.ToLower(user.Email))
+
+	if user.Email == "" {
```

Comment:
Is this check necessary since we already have regex check which empty string cannot pass?